recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   md5: d6c92acdfa253a50b2ef86a1b6443e39
 
 build:
-  number: 0
+  number: 1
```

Comment:
I know you were saying you are now convinced that this is overkill, @ocefpaf, but I think you convinced me this a good idea.

If someone checks the metadata of the current version they have, I think they should see the current maintainers list. It would at least make me wonder, if that is missing, what else is missing... Also, it may lead people to contact someone they shouldn't (if someone were ever removed).